guides/Dataset.md

```diff
@@ -940,9 +940,9 @@ Currently, there isn't a great semantic property for a Dataset to distinguish th
 <pre>
 {
   "@context": {
-    "@vocab": "http://schema.org/",
+    "@vocab": "https://schema.org/",
     "gdx": "https://geodex.org/voc/",
-    <strong>"geolink": "http://schema.geolink.org/1.0/base/main#",
+    <strong>"geolink": "https://schema.geolink.org/1.0/base/main#",
```

Comment:
Should not be changing the geolink namespace, which uses "http". See: http://schema.geolink.org/1.0/base/main.html